src/applications/static-pages/health-care-manage-benefits/components/CernerCallToAction/index.js

```diff
@@ -9,6 +9,7 @@ import LoadingIndicator from '@department-of-veterans-affairs/formation-react/Lo
 import environment from 'platform/utilities/environment';
 import { apiRequest } from 'platform/utilities/api';
 import { getCernerURL } from 'platform/utilities/constants';
+import { CERNER_FACILITY_IDS } from '../../constants';
```

Comment:
Can we store these constants with the `getCernerURL` utility? To at least start centralizing them...we could possibly do a dedicated `platform/utilities/cerner/index.js` if we suspect it might grow

Reply:
Yupp! Will do!